week-1.py

```diff
@@ -49,17 +35,19 @@ def play(self, dealerCard):
         # the 'pot' variable tracks the player's money
         
         if playerCard < dealerCard:
```

Comment:
Instead of playerCard, this should be self.lastCard. After fixing this the code works as expected.